framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/util/AmqImageStreamDeployer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.openshift.util;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShifts;
+import io.fabric8.kubernetes.api.model.KubernetesList;
+import org.kie.cloud.openshift.constants.OpenShiftConstants;
+import org.kie.cloud.openshift.resource.Project;
+
+public class AmqImageStreamDeployer {
+
+    public static void deploy(Project project) {
+        try {
+            OpenShift openShift = OpenShifts.admin(project.getName());
+            KubernetesList resourceList = openShift.lists()
+                    .inNamespace("openshift")
+                    .load(new URL(OpenShiftConstants.getAmqImageStreams()))
+                    .get();
+            resourceList.getItems().forEach(item -> {
```

Comment:
You can install AMQ image streams directly in test namespace, see https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/templates/rhpam75-prod-immutable-kieserver-amq.yaml#L378
This way you can avoid handling image stream deletion in openshift namespace.

Reply:
Also by keeping image streams in same namespace as other test image streams we can add the AMQ image stream to our image stream list, allowing us to avoid another external dependency.

Reply:
We are using AMQ GA image stream and imagese are pulled from redhat.registry.io. So we need to pulled them with secret. Same way we are pulling SSO images.

Reply:
ok